.github/ISSUE_TEMPLATE/missing-api.yaml

```
@@ -0,0 +1,57 @@
+name: Missing React Native API on Windows
+description: Report that a React Native API is missing on Windows
+title: Implement <missing API name> for <architecture>
+labels: ['enhancement', 'API: Completion']
```

```suggestion
labels: ['enhancement', 'API: Completion']
type: 'Task'
```
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#top-level-syntax

---

Can you add to the other templates too? I can't propose changes on those because the lines weren't edited in this PR. But it'd be good to have Bug/Task/Feature set.